### PR TITLE
CSFD-75

### PR DESCRIPTION
**Pull request by johnbenjamin-hmcts**
### Jira link (if applicable)
https://tools.hmcts.net/jira/browse/CSFD-75

### Change description ###
CR-84: Missing validation on auto-fill fields using Case Flags

### Checklist

- [x] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

File: infrastructure/aat.tfvars [hunk 1]
```diff
@@ -0,0 +1 @@
+redis_capacity = "3"
```
Comment (udaydenduluri33) [inline infrastructure/aat.tfvars:1]: Are you sure about this change?

Comment (johnbenjamin-hmcts) [reply infrastructure/aat.tfvars:1]: @udaydenduluri33 - It got automatically added while building

**Review by sonny-hmcts — APPROVED** [review]
LGTM

File: infrastructure/aat.tfvars (hunk 1 above)
Comment (johnbenjamin-hmcts) [reply infrastructure/aat.tfvars:1]: It was added by the PET team and pushed to master branch on 8th March

Comment (udaydenduluri33) [reply infrastructure/aat.tfvars:1]: Can you rebase it or remove it? THanks

**Review by udaydenduluri33 — APPROVED** [review]

**Pull request merged**